pdsfile/pds4file/__init__.py

```
@@ -148,9 +152,10 @@ def set_easylogger(cls):
 try:
     # Data set-specific rules are implemented as subclasses of Pds4File
     # from pdsfile_reorg.Pds4File.rules import *
-    from .rules import (cassini_iss,
-                        cassini_vims,
-                        uranus_occs_earthbased)
+    from .rules import (cassini_iss,                             # noqa: F401
+                        cassini_uvis_solarocc_beckerjarmak2023,  # noqa: F401
```

Ruff (0.12.2)

Unused `noqa` directive (unused: `F401`)

Remove unused `noqa` directive

(RUF100)